invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/communityRecordsSearch/CommunityRecordsSearchAppLayout.js

```diff
@@ -36,7 +36,7 @@ export const CommunityRecordsSearchAppLayout = ({ config, appName }) => {
               <Count
                 label={(cmp) => (
                   <Trans key="communityRecordsSearch" count={cmp}>
-                    {cmp} results found
+                    {cmp.toLocaleString(i18next.language)} results found
```

Comment:
@anikachurilova (in case you're still on it), I think this line doesn't need the `.toLocaleString(...)` since `cmp` is a component.